docs/mesos/UsageGuideUnifiedContainerizer.md

```diff
@@ -0,0 +1,121 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Using Calico Mesos and the Unified Containerizer
+The following information includes application json and information on launching tasks in a Mesos Cluster with Calico.
+
+## Prerequisites
+This guide assumes you have a cluster configured with a Mesos Master and at least
+one Mesos Agent running Calico Mesos.
+
+#### Vagrant install
```

Comment:
I don't think these should be titles. Just bullets of ways to meet the prerequisites